mozilla-release/mobile/android/base/java/org/mozilla/gecko/toolbar/ToolbarEditText.java

```diff
@@ -694,5 +702,10 @@ public void handleMessage(String event, GeckoBundle message, EventCallback callb
                 break;
         }
     }
+
+    private void setLightTheme() {
```

Comment:
You are setting nothing, the name is misleading. A better one will be `updateTheme`.